arelle/plugin/validate/EFM/__init__.py

```diff
@@ -1,806 +1,825 @@
-'''
```

Comment:
The change form unix->dos (\n -> \r\n) file endings makes it hard to see the changes.  It seems like this file should be changed back to unix (ideally with a rebase to remove the change from history).

It's been too long since I used Git on Windows to know if there's a way to fix this for everyone with `.gitattributes` or if all developers need to do something like `git config core.autocrlf input`